mappings/net/minecraft/item/ItemAbility.mapping

```diff
@@ -0,0 +1,16 @@
+CLASS net/minecraft/class_6538 net/minecraft/item/ItemAbility
```

Comment:
While this does contain the caching logic, the actual check logic is also included in this - naming the entire class "Cache" doesn't seem correct. The logic flaw sounds like a bug rather than an optimization.

Reply:
For reference, `ItemStack#canPlaceOn` and `ItemStack#canDestroy` is just a wrapper for the `class_6538#check`.